apps/docs/src/components/HomeLayout/index.tsx

```diff
@@ -17,29 +20,72 @@ interface Props {
 export const HomeLayout: FC<Props> = ({ docs, params, docsTree }) => {
 	const route = useRouter();
 
-	if (route.isFallback) {
-		return <ActivityIndicator />;
+	const node = docsTree.children?.find((node) => node.path === `/${docs}`);
+	let path = `/${docs}`;
+	if (params) {
+		for (const param of params) {
+			path += `/${param}`;
+		}
+	}
+
+	const { responsiveLevel, windowSize } = useSnapshot(dimensionState);
+
+	let scrollPaddingLeft = 20;
+	if (responsiveLevel >= 1) {
+		scrollPaddingLeft = 20;
+	} else if (windowSize.width < 1200) {
+		scrollPaddingLeft = 300;
+	} else if (windowSize.width < 1340) {
+		scrollPaddingLeft = 200;
+	} else if (windowSize.width < 1400) {
+		scrollPaddingLeft = 100;
 	}
```

Comment:
make a util function right bellow this file, but outside of the Component, maybe next to `const styles` - in case the method could be use in other place, put it under `utils/...`